src/github.com/matrix-org/go-neb/services/imgur/imgur.go

```diff
@@ -208,7 +208,7 @@ func (s *Service) text2img(query string) (*imgurGalleryImage, *imgurGalleryAlbum
 	var images []imgurGalleryImage
 	for i := 0; i < len(searchResults.Data); i++ {
 		var image imgurGalleryImage
-		if err := json.Unmarshal(searchResults.Data[i], &image); err == nil && !image.IsAlbum {
+		if err := json.Unmarshal(searchResults.Data[i], &image); err == nil && !*(image.IsAlbum) {
```

Comment:
Careful! You can't dereference and check it like this. What happens if it was `nil`? https://play.golang.org/p/oDN14Z9vqd
Instead, do:
```go
if image.IsAlbum != nil && !*(image.IsAlbum) {
    // ...
}
```